docker/smqtk_services.run_images.sh

```diff
@@ -5,14 +5,10 @@ set -e
 # a directory of images.
```

Comment:
Given my comments below, I'm not sure how necessary this script is anymore. I can see the user interaction being `IMAGE_DIR=/foo/bar docker-compose up`.